adobe/react-spectrum
# Combobox: use loading string in menu if still in loading state and no items yet

Closes #2226 

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Open a ComboBox server side filtering story.
2. Open the menu with the combobox's button before the initial api call finishes.
3. Confirm that menu now reads "Loading..." instead of "No results" before the items load.

## 🧢 Your Project:

RSP
